### PR TITLE
docs: Correct Test Database URL and testing documentation

### LINKED ISSUE #5628: Dump oevent database to opev_test
**Describe the bug**
Delete the clause in local.md to dump the oevent database into opev_test as it would mess up running local tests.

**Additional context**
Working on it.

### PR DESCRIPTION
**Pull request by prateekj117**
Fixes #5628 

#### Checklist

- [X] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [X] My branch is up-to-date with the Upstream `development` branch.
- [ ] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

File: docs/installation/local.md [hunk 1]
```diff
@@ -83,12 +83,6 @@ CREATE DATABASE oevent WITH OWNER john;
 CREATE DATABASE opev_test WITH OWNER john;
```
Comment (iamareebjamal) [inline docs/installation/local.md]: Correct the remaining doc as well

File: docs/installation/local.md [hunk 2]
```diff
@@ -80,13 +80,8 @@ For ease of development, you should create Postgres user with the same username
 ```sql
 CREATE USER john WITH PASSWORD 'start';
 CREATE DATABASE oevent WITH OWNER john;
```
Comment (iamareebjamal) [inline docs/installation/local.md]: Let's remove john and start from everywhere in the project 

Use open_event_user for user, opev_pass for password and open_event for database name

**Comment by prateekj117** [comment]
@iamareebjamal Please review.

File: docs/installation/local.md [hunk 3]
```diff
@@ -78,15 +78,9 @@ psql -d postgres
 For ease of development, you should create Postgres user with the same username as your OS account. If your OS login account is _john_, for example, you should create _john_ user in Postgres. By this, you can skip entering password when using database.
```
Comment (iamareebjamal) [inline docs/installation/local.md]: John is not gone. Please check everywhere

**Comment by iamareebjamal** [comment]
Is it removed from everywhere? All scripts and docs?

**Comment by prateekj117** [comment]
@iamareebjamal Yes, I think.

**Pull request merged**